corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -72,12 +100,25 @@ <h3 data-bind="text: $root.activeCaseType()"></h3>
                               placeholder: '{% trans "Click here to add a description" %}',
                             "></textarea>
           </td>
-          <!-- ko if: !deprecated() -->
-          <td><button data-bind="click: deprecateProperty" class="btn btn-danger btn-sm">{% trans "Deprecate Property" %}</button></td>
-          <!-- /ko -->
-          <!-- ko if: deprecated() -->
-          <td><button data-bind="click: restoreProperty" class="btn btn-default btn-sm">{% trans "Restore Property" %}</button></td>
-          <!-- /ko -->
+          {% if fhir_integration_enabled %}
+          <td>
+            <input class="form-control" data-bind="value: $data.fhirResourcePropPath, disable: removeFHIRResourcePropertyPath"></input>
+          </td>
+          {% endif %}
+          <td>
+            <!-- ko if: !deprecated() -->
+            <button data-bind="click: deprecateProperty" class="btn btn-danger btn-sm">{% trans "Deprecate Property" %}</button>
+            <!-- /ko -->
+            <!-- ko if: deprecated() -->
+            <button data-bind="click: restoreProperty" class="btn btn-default btn-sm">{% trans "Restore Property" %}</button>
+            <!-- /ko -->
+            <!-- ko if: fhirResourcePropPath() && !removeFHIRResourcePropertyPath() -->
```

Comment:
note to self: consider adding `{% if fhir_integration_enabled %}` for the buttons related to resource property path